spec/features/admin/merchants/show_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe 'As an Admin' do
+	describe 'When I visit a merchants show page' do
+		before :each do
+			@admin = User.create!(name: "Admin", address: "123 Admin cir.", city: "The provinance of holeville", state: "Ca", zip: "83845", email: "admin@gmail.com", password: "admin", role: 2)
+			@megs_shop = Merchant.create(name: "Meg's Dog Shop", address: '123 Dog Rd.', city: 'Hershey', state: 'PA', zip: 80203)
+			@brians_shop = Merchant.create(name: "Brian's Dog Shop", address: '125 Doggo St.', city: 'Denver', state: 'CO', zip: 80210)
+			@meg = @megs_shop.users.create!(name: "Meg", address: "123 merchant ave.", city: "City of Townsville", state: "Nv", zip: "39433", email: "meg@gmail.com", password: "merchant", role: 1)
+			@steve = User.create(name: 'Steve', address: '123 Street Road', city: 'City Name', state: 'CO', zip: 12345, email: 'example@example.com', password: 'password1', role: 0)
+			@order_1 = Order.create!(name: 'Meg', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17033, user: @steve)
+			@tire = @megs_shop.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+			@collar = @megs_shop.items.create(name: "Pretty Collar", description: "Your Pet will look Stunning", price: 150, image: "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.amazon.com%2FBLACK-DECKER-Collar-Tracker-Resistant%2Fdp%2FB076KVQSQG&psig=AOvVaw0FqGRwecGC3UUXzX4cHL2A&ust=1582559969577000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCODEyLmF6OcCFQAAAAAdAAAAABAE", inventory: 35)
+			@paw_pads = @megs_shop.items.create(name: "Doggie Booties", description: "Keep those paws warm", price: 80, image: "http://t0.gstatic.com/images?q=tbn%3AANd9GcRDmskkSFT5WhsbsBLHtt5sl3ilRUxjnkShsLX_ZRm7P0Aa7t0Bvw&usqp=CAc", inventory: 16)
+			@pull_toy = @brians_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
+			@rope = @brians_shop.items.create(name: "Chewy Rope", description: "Great for teething puppies", price: 50, image: "https://s7d2.scene7.com/is/image/PetSmart/5264665", inventory: 38)
+			@ball = @brians_shop.items.create(name: "Tennis Ball", description: "Great for Fetch!", price: 45, image: "https://images-na.ssl-images-amazon.com/images/I/51QTU9nQCHL._AC_.jpg", inventory: 23)
+			@order_1.item_orders.create!(item: @tire, price: @tire.price, quantity: 2)
```

Comment:
We are accumulating lots of goodies for generous seeds file.

A compilation project in the future.